go/stgin/go.mod

```diff
@@ -0,0 +1,7 @@
+module main
+
+go 1.18
+
+require github.com/AminMal/stgin v1.1.0
+
+require github.com/AminMal/slogger v1.0.0 // indirect
```

Comment:
why both ? is there no a require only module that require both ?

Reply:
Unfortunately not for now. That's the logging lib. But I'll make sure to take a look on how to do it in future stgin releases.